test/test_unit_keystore_adapter.py

```diff
@@ -0,0 +1,43 @@
+# Builtin
+import unittest
+from unittest.mock import patch, call
+# Third-Party
+import keyring
+# Internal
+from conjur.wrapper import KeystoreAdapter
+
+CONJUR_URL = "https://conjur-hostname.com"
+
+
+class KeystoreAdapterTest(unittest.TestCase):
+    @patch.object(keyring, 'delete_password')
+    def test_delete_password_calls_keyring_delete_password(self, mock_keyring):
+        KeystoreAdapter.delete_password(CONJUR_URL)
+        calls = [
+            call(CONJUR_URL, "machine"),
```

Comment:
there are constants for machine, login, and password. Let's change them so that if we one day change the names, our tests don't break as a result

Reply:
Changed